language/transaction-builder-generator/src/python3.rs

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use libra_types::transaction::{ArgumentABI, ScriptABI, TypeArgumentABI};
+use move_core_types::language_storage::TypeTag;
+
+use std::io::{Result, Write};
+
+pub fn output(out: &mut dyn Write, abis: &[ScriptABI]) -> Result<()> {
+    output_preambule(out)?;
+    for abi in abis {
+        output_builder(out, abi)?;
+    }
+    Ok(())
+}
+
+fn output_preambule(out: &mut dyn Write) -> Result<()> {
```

Comment:
```suggestion
fn output_preamble(out: &mut dyn Write) -> Result<()> {
```

Reply:
haha that's the right spelling but not in English :)